Add React autosuggest components

moroshko opened the pull request:
First of all, thanks for a great project!

I tried to follow the README to add a comparison for React autosuggest components, but the API call doesn't seem to take into consideration the updated `data.json`. 

I'm expecting to see "React autosuggest components" under "Categories:", but no luck.

What am I missing?

moroshko committed:
Add React components

dmitry-zaets commented:
@moroshko thanks for a PR!

API server takes `data.json` from master branch, that's why you was not able to see it.
So changes will apply only after a merge. (I'm working on open-sourcing server, so you will be able to test locally)

I think it is a good idea to add `React Components` to the chart!
But `React autosuggest components` is a bit too concrete.
Is it ok to change category name and label to `React Components`? Then it will allow comparing all components in general?

Also, you can add `React autosuggest components` to https://github.com/StarRatio/star-ratio/blob/master/src/components/LinksList/index.js

moroshko commented:
Happy to change to "React components". I updated the PR. Also added few more React components.

I also noticed that on the "Compare" page the repos appear in a non deterministic order (the order in which requests come back?). I'd expect them to appear in the oder they are defined [here](https://github.com/StarRatio/star-ratio/blob/1230e2b4a22e2d1f48bd034f1e93d3df459a0350/src/components/LinksList/index.js#L9-L28).

dmitry-zaets commented:
Thanks for PR!

That's a bug, I will fix it.

dmitry-zaets commented:
Fixed!
